fenix/app/src/main/java/org/mozilla/fenix/components/metrics/GrowthActivatedUserWorker.kt

```diff
@@ -0,0 +1,74 @@
+package org.mozilla.fenix.components.metrics
+
+import android.content.Context
+import androidx.work.ExistingWorkPolicy
+import androidx.work.OneTimeWorkRequest
+import androidx.work.WorkManager
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import mozilla.components.support.utils.ext.getPackageInfoCompat
+import org.mozilla.fenix.ext.metrics
+import org.mozilla.fenix.ext.settings
+import java.util.concurrent.TimeUnit
+
+/**
+ * Worker that will send the User Activated event at the end of the first week.
+ */
+class GrowthActivatedUserWorker(
+    context: Context,
+    workerParameters: WorkerParameters,
+) : Worker(context, workerParameters) {
+
+    override fun doWork(): Result {
+        val settings = applicationContext.settings()
+
+        if (!System.currentTimeMillis().isEndOfFirstWeek(applicationContext) ||
+            settings.growthUserActivatedSent
+        ) {
+            return Result.success()
+        }
+
+        applicationContext.metrics.track(Event.GrowthData.UserActivated(fromSearch = false))
+
+        return Result.success()
+    }
```

Comment:
Would you mind filing a follow-up to add the rest of the metrics that are currently tracked through `MetricsMiddleware` here as well? This seems like a useful fallback for all of those